modules/apps/headless/headless-builder/headless-builder-api/src/main/java/com/liferay/headless/builder/application/APIApplication.java

```diff
@@ -57,6 +59,12 @@ public enum Scope {
 
 	}
 
+	public interface Filter {
+
+		public String getODataFilter();
```

Comment:
I think this should end with `String`. Something like `getODataFilterString` or `getODataString`.

Reply:
Changed